addons/sale/sale.py

```diff
@@ -417,6 +418,20 @@ def _create_analytic_account(self, prefix=None):
             })
             order.project_id = analytic
 
+    @api.multi
+    def _get_taxes_by_group(self):
+        self.ensure_one()
+        Tax = self.env['account.tax']
+        res = {}
+        for line in self.order_line:
+            taxes = line.tax_id.compute_all(line.price_unit, quantity=line.product_uom_qty)['taxes']
+            for tax in taxes:
+                group = Tax.browse(tax['id']).tax_group_id
```

Comment:
Rebrowsing the each tax on each line is a bad idea as it is already browsed with the line